site/content/blog/2022-10-01-zap-and-hacktoberfest/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "ZAP and Hacktoberfest"
+summary: >
+    ZAP is partcipating in Hacktoberfest 2022.
+images:
+- https://www.zaproxy.org/site/blog/2022-10-01-zap-and-hacktoberfest/images/Hfest-Badge-2-Color-Void@2x.png
+type: post
+tags:
+- blog
+- hacktoberfest
+date: "2022-10-01"
+addSocialPreview: true
+authors:
+- arkaprabha
+---
+![Hacktoberfest X ZAP](images/zap_Hacktoberfest.png)
+
+We are happy to announce that ZAP is partcipating in Hacktoberfest 2022. Hacktoberfest 2022 is a month-long celebration of open source software presented by [DigitalOcean](https://digitalocean.com/), [Appwrite](https://appwrite.io/) and [Docker](https://docker.com/). Hacktoberfest is open to everyone in our community! You can sign up anytime between October 1 and October 31.
```

Comment:
- Typo in participating.
 - Link to the official site instead.

Reply:
official site? of hacktoberfest?

Reply:
typo fixed

Reply:
Yes but link to that site instead of the others, we don't need to do the marketing, already done in the site.